src/containers/webuild-day/TheProgram/LeftSide.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const LeftSide = () => {
+  return (
+    <div className="left-side">
+      <h3 className="text-30px font-extrabold text-foreground leading-tight">
+        Details
+      </h3>
+      <div className="mt-8 text-base leading-tight">
+        <div className="font-bold">Location</div>
+        <p className="mt-1">Block 71 Saigon</p>
+        <div className="font-bold mt-2">Date</div>
+        <p className="mt-1">December 12th, 2020</p>
+        <div className="font-bold mt-2">Time</div>
+        <p className="mt-1">Morning Session: 9am - 12.30pm</p>
+        <p className="mt-1">Evening Session: 2.30pm - 6pm</p>
```

Comment:
Should `p` tag have the color of `#FEFCF8` like the design? Can you get it fixed in other places if have?